webapp/templates/partials/scoreboard_table.html.twig

```diff
@@ -122,7 +142,7 @@
             {% set color = score.team.category.color %}
         {% endif %}
         <tr class="{{ classes | join(' ') }}" id="team:{{ score.team.teamid }}">
-            <td class="scorepl">
+            <td class="scorepl{{awardColor}}">
```

Comment:
Maybe always add a space to the class here so we don't have to add it to the variable? Does not really hurt.

Reply:
The default setting is `awardColor=''`, so its length is 0, actually if a team does not win any award, there will be no spaces in this class.